spec/mirage/client/client_spec.rb

```diff
@@ -17,7 +17,7 @@
       mirage_url = "http://url.for.mirage"
       Client.new(mirage_url).url.should == mirage_url
 
-      Client.new(:url => mirage_url).url.should == mirage_url
+      Client.new(:url => mirage_url, :port => 9001).url.should == mirage_url
```

Comment:
I'm confused with this port here. This test doesn't test port at all, so it seems to be unnecessary and left to set it to default value. You test configuration with port in the tests below.